docs/document/i18n/ja/docusaurus-plugin-content-docs/current/getting-started/quick-start.md

```diff
@@ -1,50 +1,57 @@
 以下のような kintone ライクな UI を簡単に作成できるので、フォーム部品を自作したい場合に便利にお使いいただけます。
 
 ### Desktop 用コンポーネント例
 
-![desktop components](assets/desktop_components.png)
+![desktop components](/img/desktop_components.png)
 
 ### Mobile 用コンポーネント例
 
-![mobile components](assets/mobile_components.png)
+![mobile components](/img/mobile_components.png)
 
 ## 下準備：アプリ作成
 
-ここでは既定の kintone 一覧画面のヘッダーメニュースペースに Button コンポーネントを配置します。<br>
+ここでは既定の kintone 一覧画面のヘッダーメニュースペースに Button コンポーネントを配置します。<br/>
 まず kintone UI Component を設置するアプリを用意します。
 
-1. 空のアプリを作成します。（[アプリをはじめから作成する](https://jp.cybozu.help/k/ja/user/create_app/tutorial.html)）<br>
+1. 空のアプリを作成します。（[アプリをはじめから作成する](https://jp.cybozu.help/k/ja/user/create_app/tutorial.html)）<br/>
 特にフィールドの設置は必要ありません。
 
 ## 導入と実装方法
 
-kintone UI Component では UMD と CDN と npm パッケージを用意しています。<br>
-ブラウザ環境や Node.js 環境など開発環境に合わせていずれかを選択して読み込み、実装してください。<br>
+kintone UI Component では UMD と CDN と npm パッケージを用意しています。<br/>
+ブラウザ環境や Node.js 環境など開発環境に合わせていずれかを選択して読み込み、実装してください。<br/>
 本記事では、それぞれの導入・実装方法を紹介します。
 
-> プロジェクト用の kintone UI Component を最新バージョンにアップデートして利用することを推奨しています。
+:::tip
+プロジェクト用の kintone UI Component を最新バージョンにアップデートして利用することを推奨しています。
+:::
 
-> 各コンポーネントのリファレンスについてはサイドバーの Components カテゴリよりご確認ください。<br>
-> （順次対応コンポーネントも追加予定です。）
+:::tip
+各コンポーネントのリファレンスについてはサイドバーの Components カテゴリよりご確認ください。<br/>
+（順次対応コンポーネントも追加予定です。）
+:::
 
 ### UMD を利用する
 
-> v1.4.0 以降のバージョンを利用する場合は、コンポーネントを呼び出す際に Kuc オブジェクトの代わりに Kucs["1.x.x"] を使ってバージョンを指定してください。（ex. `new Kucs["1.4.0"].Button()`）<br>
-> レンダリングされたコンポーネントのタグとクラス名にはバージョン番号が含まれます。<br>
-> グローバルオブジェクトとして Kuc を使うこともできますが、2つ以上の kuc.min.js を kintone カスタマイズやプラグインに読み込む場合はバージョンコンフリクトが起きる可能性があるのでご注意ください。この際、Kuc オブジェクトは最後に読み込まれた kuc.min.js を参照します。<br>
-> システム上に kuc.min.js が 1つしかない、もしくは最後に読み込まれた kuc.min.js の利用で問題ない場合は、Kuc オブジェクトを利用いただいて問題ありません。`const Kuc = Kucs['1.x.x'];` の行を削除してください。<br>
-> v1.3.2 以前のバージョンを利用する場合は、Kuc をグローバルオブジェクトとして使ってください。2つ以上の kuc.min.js を kintone カスタマイズやプラグインに追加すると、バージョンコンフリクト問題が起きるのでご注意ください。
-> 詳しくは、[Version conflicts issue and solution](../guides/version-conflicts-issue-solution) 記事をご確認ください。
+:::tip
+v1.4.0 以降のバージョンを利用する場合は、コンポーネントを呼び出す際に Kuc オブジェクトの代わりに Kucs["1.x.x"] を使ってバージョンを指定してください。（ex. `new Kucs["1.4.0"].Button()`）<br/>
+レンダリングされたコンポーネントのタグとクラス名にはバージョン番号が含まれます。<br/>
+グローバルオブジェクトとして Kuc を使うこともできますが、2つ以上の kuc.min.js を kintone カスタマイズやプラグインに読み込む場合はバージョンコンフリクトが起きる可能性があるのでご注意ください。この際、Kuc オブジェクトは最後に読み込まれた kuc.min.js を参照します。<br/>
+システム上に kuc.min.js が 1つしかない、もしくは最後に読み込まれた kuc.min.js の利用で問題ない場合は、Kuc オブジェクトを利用いただいて問題ありません。`const Kuc = Kucs['1.x.x'];` の行を削除してください。<br/>
+v1.3.2 以前のバージョンを利用する場合は、Kuc をグローバルオブジェクトとして使ってください。2つ以上の kuc.min.js を kintone カスタマイズやプラグインに追加すると、バージョンコンフリクト問題が起きるのでご注意ください。
+詳しくは、[Version conflicts issue and solution](../guides/version-conflicts-issue-solution) 記事をご確認ください。
+
+1. kintone UI Component リポジトリ内の[各バージョン Release 欄](https://github.com/kintone-labs/kintone-ui-component/releases)に添付のアーカイブフォルダ（kintone-ui-component-{version}.tgz）を解凍し、以下のファイルを kintone アプリ設定の `JavaScript / CSS でカスタマイズ`にて指定します。
```

Comment:
Please remove the needless lines.
```suggestion
```

Reply:
There is exist this line in version 1.11.0 at here: https://kintone-ui-component.netlify.app/docs/ja/getting-started/quick-start
https://raw.githubusercontent.com/kintone-labs/kintone-ui-component/master/docs/document/website/translated_docs/ja/version-1.11.0/getting-started/quick-start.md
Why do we remove it?

Reply:
These lines are in the tips section and I think they are redundant.
Because there is the same message below.

Reply:
thanks for your kind reply, i understand and fixed